### PR TITLE
Added marvel and icong

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by dhardy92** [comment]
does this permit build for HTC wildfire S fully fonctional ?

**Comment by lissyx** [comment]
Please open a bug on bugzilla and reference it in the name of the PR

**Comment by vigoos** [comment]
I think I'm having problems with this build. I'm closing pull request and trying to fix the error.

**Comment by vigoos** [comment]
UPDATE 1- marvel build should work fine to the best of my knowledge.

**Comment by dhardy92** [comment]
Just created a a bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1165716

File: marvel.xml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  <project path="prebuilt" name="platform/prebuilt" />
+  <project path="system/bluetooth" name="platform/system/bluetooth" />
+  <project path="system/core" name="platform/system/core" />
+  <project path="system/extras" name="platform/system/extras" />
+  <project path="system/media" name="platform/system/media" />
+  <project path="system/netd" name="platform/system/netd" />
+  <project path="system/vold" name="platform/system/vold" />
+
+ -  <!-- Wildfire S things -->
+-<project path="device/htc/marvel" name="b2g_device_htc_marvel" remote="izi501_b2g" revision="master" />
```
Comment (lissyx) [inline marvel.xml]: You have extra '-' just before the <project, this is probably not what you want.

**Comment by vigoos** [comment]
Updated to KitKat base

**Pull request closed without merging**